docs/docs/20-quickstart.md

```diff
@@ -261,6 +265,8 @@ all three Argo CD `Application`s have not yet synced because they're not
 configured to do so automatically, and in fact, the branches referenced by their
 `targetRevision` fields do not even exist yet.
 
+![Argo-dashboard-screenshot](../static/img/argo-dashboard-2.png)
```

Comment:
Compared to the prior two screenshots, I am more ok with this one because its more than just a blank page and gives the readers a quick and easy way to assert the success of previous steps.